q2_feature_table/plugin_setup.py

```diff
@@ -72,6 +88,12 @@
     parameters={},
     outputs=[
         ('merged_table', FeatureTable[Frequency])],
+    input_descriptions={
+        'table1': 'The first feature table to be merged.',
+        'table2': 'The second feature table to be merged.',
+    },
+    parameter_descriptions={},
+    output_descriptions={'merged_table': 'The resulting feature table.'},
```

Comment:
*The resulting merged feature table containing all features from `table1` and `table2`.*